scripts/ci/pre_commit/pre_commit_update_providers_dependencies.py

```diff
@@ -175,7 +175,7 @@ def check_if_different_provider_used(file_path: Path) -> None:
         if imported_provider is not None and imported_provider not in ALL_PROVIDERS:
             warnings.append(f"The provider {imported_provider} from {file_path} cannot be found.")
         elif imported_provider and file_provider != imported_provider:
-            ALL_DEPENDENCIES[file_provider][CROSS_PROVIDERS_DEPS].append(imported_provider)
+            ALL_DEPENDENCIES[file_provider]["cross-providers-deps"].append(imported_provider)
```

Comment:
Is there a reason we're making this change?

Reply:
Main reason was that when I added "excluded-python-versions" I realized that extracting those into constants makes very little sense (and I would have to add another EXCLUDED_PYTHON_VERSIONS constant for consistency, so I decided to consistently get rid of the constants :).